wordpress-mobile/WordPress-iOS
# UL&S Fixes some Google Signup tracking issues.

WPAuth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/471

Fixes some Google Signup tracking issues.

## Testing:

1. Tap “Continue with WordPress.com”
2. Tap “Continue with Google”
3. Select your Google credentials that have no matching WP.com account
4. When the “Sign Up” screen comes up, verify:

🔵 Tracked: unified_login_step <flow: google_signup, source: default, step: start>

5. Tap on “Create account” and verify:

🔵 Tracked: unified_login_interaction <click: submit, flow: google_signup, source: default, step: start>
🔵 Tracked: unified_login_interaction <click: create_account, flow: google_signup, source: default, step: start>
🔵 Tracked: unified_login_step <flow: google_signup, source: default, step: success>

6. Click “Done”
7. Click “Not right now” and verify

🔵 Tracked: unified_login_interaction <click: dismiss, flow: google_signup, source: default, step: success>

## PR submission checklist:
